README.md

```diff
@@ -18,12 +18,17 @@
    Hugo Static Site Generator v0.78.2/extended darwin/amd64 BuildDate: unknown
    ```
 
-2. [Fork](https://github.com/rakutentech/engineering-blog/fork) & Clone this repository
-3. Run the local server with:
+2. [Fork](https://github.com/rakutentech/engineering-blog/fork)
+3. Clone your repository and initialize its submodules
+   ```
+   > git clone <YOUR GIT REPO CLONE URL>
```

Comment:
Should we instruct to do `git clone --recurse-submodules` <URL>` instead of 2 separate commands?

Reply:
Probably better yes